vm-setup/roles/libvirt/templates/baremetalvm.xml.j2

```diff
@@ -18,6 +19,12 @@
     <boot dev='network'/>
     <bootmenu enable='no'/>
   </os>
+{% else %}
+<os firmware='efi'>
```

Comment:
We can probably use it for all architectures now that we no longer support RHEL 7.